docs/license.rst

```diff
@@ -1,6 +1,13 @@
 License
 =======
 
+.. Sadly, I cannot figure out a way to read the LICENSE file and
```

Comment:
😄 

Yeah, fair enough.  Just a knee-jerk reaction to having the same text in multiple places.

For example, in `LICENSE`, we actually include the MPICH license, too (assumedly because of ROMIO).  I'm guessing that should go here, too.

Do we also need to cite all the OMPI + PMIX + PRTE copyrights here?

Reply:
Tough question - you'd need to include libevent and hwloc as well, if you go that route. I'd say you don't need the text and could just refer to them as (a) those are standalone projects and (b) people can go read those license files, especially if you provide a link to them.

Reply:
Given how we build libevent, hwloc, pmix, and prrte (as stand-alone packages that are installed in the same location), I think a license by URL link is a reasonable solution.